src/mlagility/analysis/analysis.py

```diff
@@ -20,8 +20,8 @@
 import mlagility.analysis.status as status
 import mlagility.analysis.util as util
 import mlagility.common.labels as labels
-from mlagility.api import benchit
-from mlagility import filesystem
+from mlagility.api.api import benchit
```

Comment:
It is quite confusing that we can import things from `mlagility.api.api` and `mlagility.api`. We should probably move things around so that is never the case.